spec/presenters/keyword_presenter_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe KeywordPresenter do
+  describe '#keyword' do
```

Comment:
```suggestion
  describe '#matching_adword_urls' do
```

It seems that we are not testing the `keyword` attribute, but instead, the method `matching_adword_urls`.